readme/BREAKING_CHANGES.markdown

```diff
@@ -580,4 +580,30 @@ API to subscribe to `imageSelectorCoverImageAtom`.
 channel. The `Liferay.State` API is a better fit for modules that wish to
 coordinate at a distance in this way, and it does so in a type-safe manner.
 
+---------------------------------------
+
+### OAuth2 feature Token Instrospection value has changed
+- **Date:** 2021-May-04
+- **JIRA Ticket:** [LPS-131573](https://issues.liferay.com/browse/LPS-131573)
+
+#### What changed?
+
+The feature value for Token Introspection at OAuth2 has changed from 
+`token_introspection` to `token.introspection` due to standarize 
+OAuth2 constants.
+
+#### Who is affected?
+
+This affects any development that uses the Token Introspection feature value.
```

Comment:
We might want to extend the impact text with what @stian-sigvartsen wrote: any usages where OAuth2Application is added programmatically through `OAuth2ApplicationLocalService` with a `featuresList` containing `token_introspection`.

Reply:
"added or inspected programmatically". @martamedio don't worry about this, I can tweak it in my review